### ISSUE TITLE
Hi, is it still working?

### ISSUE THREAD

**Issue by Thyoity** [trickpattyFH20/server-lol-chat#1]
Because I logged in to my account, but it shows nothing, no friends in the online/offline list. Tried with a wrong credential too and I think the authentication is working fine as it should.

**Comment by trickpattyFH20** [comment]
I just tested a login and nothing seems to be broken for me, online and offline lists are both showing. http://league.chat always uses the latest build of server-lol-chat and that is also working.

There could still be a bug though :) If you find any error logs let me know! Did you see any build errors? Maybe something with an npm dependency?

**Comment by Thyoity** [comment]
See here my console:
http://image.prntscr.com/image/695da4da6a1343bebc9c0f3f53e67206.png

I didn't install in my environment yet. But I tested directly in league.chat.

**Comment by trickpattyFH20** [comment]
very interesting. after briefly reading about that error message [here](http://stackoverflow.com/questions/31358446/failed-to-clear-temp-storage) it seems like it could be a client side issue, so lets try to rule that out first. would you mind trying it in a different browser and letting me know what happens?

**Comment by trickpattyFH20** [comment]
very interesting. after briefly reading about that error message it seems like it could be a client side issue, so lets try to rule that out first. would you mind trying it in a different browser and letting me know what happens?

**Comment by Thyoity** [comment]
Of course! I also tried with Mozilla Firefox and Internet Explorer and in these browsers there are no error messages. But even in these browsers, they are not showing my LoL friendlist, maybe there are something to do with my LoL region? I'm on BR server. I tried with several accounts, and none of them is showing the friendlist. But I know that the authentication is working, because when I propositally enter a wrong credential, it shows me a XMPP error.

**Comment by Thyoity** [comment]
Of course! I also have tested in Firefox and Internet and in these browsers league.chat is not outputting errors. But even in these browsers, my friendlist is not showing neither in Online or Offline tabs. Perhaps its something to do with my LoL region? I'm from BR server. I tried with 3 different accounts, and none of them is working as it should. But I know that the authentication is working correctly, because if I propositally fill the login/password fields with incorrect credentials, it shows me a XMPP error.

**Comment by trickpattyFH20** [comment]
aha! so I just pinged the BR server `$ ping chat.br.lol.riotgames.com` and I receive a request timeout:

```
PING chat.br.lol.riotgames.com (): 56 data bytes
Request timeout for icmp_seq 0
```

A good response looks like this:

```
PING chat.na2.lol.riotgames.com (): 56 data bytes
64 bytes from 192.64.174.69: icmp_seq=0 ttl=51 time=75.705 ms
```

I will do some more investigation on the forums tonight, maybe they renamed the Brazil chat server

Thanks for the find! @Thyoity

**Comment by trickpattyFH20** [comment]
I'm worried that the ISP's in north america could actually be blocking the connection here, because the league.chat server is an AWS server in the United States, and I am receiving a request timeout from multiple ip's based here. But Riot says the Brazil servers are fine, and it looks like the domain didn't change.

@Thyoity It seems like the Brazil League servers are fine, you're still able to log in using the official League client right?

If you'd like to help confirm this, try building the league-chat project locally and connecting again, I'm guessing it will work fine and you'll be able to login. If this is the case, I will consider finding a VPS in brazil to balance requests from there.

Here is the League Chat repo if you'd like to try a local build, it should be pretty easy to set up: https://github.com/trickpattyFH20/league-chat

**Comment by Thyoity** [comment]
That is weird, because here is my LoL configuration file:

```
host=prod.br.lol.riotgames.com,prod.br.lol.riotgames.com
xmpp_server_url=chat.br.lol.riotgames.com
lq_uri=https://lq.br.lol.riotgames.com
storyPageURL=http://br.leagueoflegends.com/story
rssStatusURLs=null
regionTag=br
lobbyLandingURL=https://frontpage.br.leagueoflegends.com/$localeCode/client/landing
ladderURL= http://competitive.br.leagueoflegends.com/pt-br/ladders
loadModuleChampionDetail=true
featuredGamesURL=http://spectator.br.lol.riotgames.com:80/observer-mode/rest/featured 
platformId=BR1
ekg_uri=https://ekg.riotgames.com
riotDataServiceDataSendProbability=1.0
```

If you see the "xmpp_server_url" param, it points to: "chat.br.lol.riotgames.com". I tried to login to my lol account right now, and I successfully connected to the chat.
I also tried to ping to "chat.br.lol.riotgames.com" and I received request timeout too. With the na2 domain, it worked fine. So in short, I don't think the BR chat server is experiencing issues because in the client, everything is working fine.

**Comment by Thyoity** [comment]
Maybe the BR server are blocking ping requests.

Yes, I will try to build a local server. And will post the result here.

**Comment by trickpattyFH20** [comment]
seems like other regions have weird bugs also, not just with this repo but with chat in general. I'm not even able to make a test account on EUW and log in to the official iOS LoL friends app. have you tried any NA accounts yet? @Thyoity

**Comment by Thyoity** [comment]
Sorry, I didn't have time to test it yet. But I'll do it today. Going to post my conclusions here soon.
